.github/workflows/ci.yml

```
@@ -10,10 +10,25 @@ concurrency:
   group: ${{ github.workflow }}
 
 jobs:
+  build-uefi-sb:
+    uses: ./.github/workflows/build.yml
+    with:
+      cacheless: false
+      kas-path: meta-dts/kas-uefi-sb.yml
+  move-build-artifacts:
+    needs: build-uefi-sb
+    runs-on:
+      labels: dts-builder
+    steps:
+      - name: Move UEFI SB build artifacts
+        shell: bash
+        run: |
+          mv build build-sb
```

```suggestion
        run: mv build build-sb
```

no need for `shell: bash` if you don't need anything from bash.

---

https://github.com/Dasharo/meta-dts/compare/289298c32455596b46e88b8fb1dec5ba4a163843..15a67a9fab3df44324e6d5bdd2d30f2eac37f917